x-pack/plugins/security_solution/cypress/e2e/exceptions/add_edit_flyout/flyout_validation.cy.ts

```diff
@@ -93,19 +92,18 @@ describe('Exceptions flyout', { testIsolation: false }, () => {
         })
       )
     );
+    login();
```

Comment:
why do we need `login()` here as well? I see that you moved it into `beforeEach`

Reply:
Seems redundant indeed, will remove.